build/config.props

```diff
@@ -32,18 +32,16 @@
     <VsTargetMajorVersion>$([MSBuild]::Add(11, $(MajorNuGetVersion)))</VsTargetMajorVersion>
     <VsTargetBranch>master</VsTargetBranch>
     <VsTargetChannel>int.$(VsTargetBranch)</VsTargetChannel>
+    <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetBranch>
+    <VsTargetChannel Condition="'$(IsEscrowMode)' == 'true'">int.d$(VsTargetMajorVersion).$(MinorNuGetVersion)</VsTargetChannel>
+
     <!-- when LockSDKVersion is true,  it will use the specific version in CliVersionForBuilding for building, and specific version in OverrideCliBranchForTesting for testing,  -->
     <!-- when LockSDKVersion is not true, it will ignore the properties and just use the latest version for the channel specified in CliVersionForBuilding for building, CliBranchForTesting for testing -->
-    <LockSDKVersion>true</LockSDKVersion>
+    <LockSDKVersion></LockSDKVersion>
     <OverrideCliBranchForTesting Condition="'$(LockSDKVersion)' == ''"></OverrideCliBranchForTesting>
```

Comment:
See [this](https://github.com/NuGet/NuGet.Client/pull/3161/files#diff-d81258fd0247d8cabf486cb289163d5eR39).
```XML
Condition="'$(LockSDKVersion)' != 'true'"
```

Reply:
Thanks. Changed.